### PR TITLE
[FIX] remove translation message

### PR DESCRIPTION
**Pull request by fmdl**
@sylvain-garancher @elicoidal

### CODE REVIEW

**Review by elicoidal — APPROVED** [review]
LGTM

File: odoo_sentinel/__init__.py [hunk 1]
```diff
@@ -904,7 +904,7 @@ def main():
         '-p', '--profile', dest='profile', default='sentinel',
         help='OdooRPC profile to use.')
     parser.add_argument(
-        '-c', '--config', dest='config_file', default='~/.odoorpcrc',
```
Comment (Garamotte) [inline odoo_sentinel/__init__.py:907]: Why did you change the configuration file's location ?
https://pythonhosted.org/OdooRPC/tuto_session.html?highlight=odoorpcrc

**Review by Garamotte — CHANGES_REQUESTED** [review]

**Comment by fmdl** [comment]
Pas ce que si il y a le . sur mon mac ca ne marche pas. 

Florent de Labarre

**Comment by fmdl** [comment]
et le problème c'est lors d'un pip install il n'y a pas le fichier.

**Comment by yvaucher** [comment]
@fmdl the default location should remains a hidden file, it make more sense.
I doubt pip install will create the `.odoorpcrc` file for you.

You should be able to use the `-c` parameter to select a specific configuration file.

**Comment by yvaucher** [comment]
Just saw it is overseeded by https://github.com/OCA/odoo-sentinel/pull/15

Thus I will close this PR
